Add data_usage method

JureZmrzlikar opened the pull request.

mstajdohar commented:
Approved. Rebase and merge.

dblenkus commented on resdk/tests/unit/test_resolwe.py:
```diff
@@ -464,6 +464,32 @@ def test_good_response(self, resolwe_mock, requests_mock, os_mock, open_mock):
 
+class TestDataUsage(unittest.TestCase):
+
+    @unittest.skipIf(six.PY2, "Skip in case of python2.x")
```
Why this doesn't work in py2?

dblenkus commented on resdk/tests/unit/test_resolwe.py:
```diff
@@ -464,6 +464,32 @@ def test_good_response(self, resolwe_mock, requests_mock, os_mock, open_mock):
+class TestDataUsage(unittest.TestCase):
+
+    @unittest.skipIf(six.PY2, "Skip in case of python2.x")
+    def test_data_usage(self):
```
I doubt it is worth having such test as it is testing a really small part of whole functionality. It would be much better to write an e2e test that actually makes request to backend.

mstajdohar commented:
Do we support sorting data usage on the server?

JureZmrzlikar commented:
@dblenkus I've added e2e tests and removed the silly unit test.
@mstajdohar  I've added support for sorting on the backend and also enable this in SDK. As an example check the `test_ordering` e2e test.

mstajdohar commented:
Let's merge this. Thanks
